pkgs/gohugoio/hugo/hugo-extended/registry.yaml

```diff
@@ -0,0 +1,54 @@
+packages:
+  - type: github_release
+    repo_owner: gohugoio
+    repo_name: hugo
+    name: gohugoio/hugo/hugo-extended
+    asset: hugo_extended_{{trimV .Version}}_{{.OS}}-{{.Arch}}.{{.Format}}
+    description: The world’s fastest framework for building websites
+    supported_envs:
+      - linux
+      - amd64
+      - darwin
+    files:
+      - name: hugo
+    format: tar.gz
+    checksum:
+      type: github_release
+      asset: hugo_{{trimV .Version}}_checksums.txt
+      file_format: regexp
+      algorithm: sha256
+      pattern:
+        checksum: ^(\b[A-Fa-f0-9]{64}\b)
+        file: "^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
+    version_constraint: semver(">= 0.103.0")
+    overrides:
+      - goos: windows
+        format: zip
+      - goos: darwin
+        asset: hugo_extended_{{trimV .Version}}_{{.OS}}-universal.tar.gz
+    version_overrides:
+      - version_constraint: semver("< 0.102.0")
+        supported_envs:
+          - amd64
+          - darwin
+        replacements: *hugo_replacements_1
+        overrides:
+          - goos: windows
+            format: zip
+      - version_constraint: semver(">= 0.102.0")
+        overrides:
+          - goos: windows
+            format: zip
+          - goos: darwin
+            asset: hugo_extended_{{trimV .Version}}_{{.OS}}-universal.tar.gz
+        replacements: &hugo_replacements_1
```

Comment:
The anchor is conflicted.
https://github.com/aquaproj/aqua-registry/blob/f0e1e59165bcc3adf6abf49b6f58abfb37352b2f/pkgs/gohugoio/hugo/registry.yaml#L31